tests/integration/linux_bridge_test.py

```diff
@@ -295,6 +297,19 @@ def test_linux_bridge_over_bond_over_slave_in_one_transaction(bond0):
     assertlib.assert_absent(bridge_name)
 
 
+def test_rollback_for_linux_bridge():
+    current_state = libnmstate.show()
+    bridge_name = TEST_BRIDGE0
+    bridge_state = _create_bridge_subtree_config(("invalid_port",))
+    with pytest.raises(NmstateVerificationError):
+        with linux_bridge(bridge_name, bridge_state) as desired_state:
+            libnmstate.apply(desired_state)
+
+    time.sleep(5)  # Give some time for NetworkManager to rollback
+    current_state_after_apply = libnmstate.show()
+    assert current_state == current_state_after_apply
```

Comment:
It's not current state anymore. Please rename it to `original_state` or something alike.